roles/ceph-common/tasks/configure_cluster_name.yml

```diff
@@ -19,34 +19,36 @@
 # - Jewel from latest Canonical 16.04 distro
 # - All previous versions from Canonical
 # - Infernalis from ceph.com
-- name: check /etc/default/ceph exist
-  stat:
-    path: /etc/default/ceph
-  register: etc_default_ceph
-  check_mode: no
-  when:
-    - ansible_os_family == "Debian"
+- name: debian - configure cluster name
```

Comment:
`ansible_os_family == "Debian"` doesn't mean exclusively debian, it can be ubuntu as well

should be probably renamed to something like => "debian based system" / "debian based distribution" / ...

Reply:
In the attempt to keep the task names shorter, what about "debian family"

Reply:
Hum "debian family" could interpreted as debian version. So I'd stick with ""debian based systems". Thanks